cdap-validation/bin/ambari.py

```diff
@@ -0,0 +1,87 @@
+import urllib2
+import json
+import re
+import glob, os
+import test_helpers as helpers
+
+# Start Ambari configuration retrieval process
+def ambari_commands(host, subdir, base, cluster_info):
+    ## get ambari configs
+    manager = 'ambari' 
+    cu = get_config_urls(host, cluster_info)
+    #print 'cu=%s' % (cu)
+    config_url_href = cu.read()
+    #print 'configurlref=%s' % (config_url_href)
+
+    ## extract the ambari config urls
+    data = json.loads(config_url_href)
+    config_urls = [ item['href'] for item in  data['items'] ]
+
+    ## iterate through ambari config urls, and run api requests against those
+    ##   This creates config files in a subdirectory
+    get_configs(host, config_urls, subdir, cluster_info)
+
+    # process tmp configs and store results
+    ambari_stored_configs = base[manager]['stored_results']
+    store_results(subdir, ambari_stored_configs, manager)
+
+# get config urls
+def get_config_urls(host, cluster_info):
+    cluster = cluster_info['cluster']  
+    append = '/clusters/' + cluster + '/configurations' 
+    url = host + append
+    print 'url=%s' % (url)
+    config_urls = helpers.run_request(url, cluster_info)
+    return config_urls
+
+# get configs through config urls
+def get_configs(host, urls, subdir, cluster_info):
+    for uri in urls:
+        # extract service file name
+        m = re.search('\?type=(.+?)&tag', uri)
+        file = m.group(1)
+
+        # run api config retrieval command and write result to file
+        helpers.get_config_and_write(uri, subdir, file, cluster_info)
+
+# process tmp configs and store results
+def store_results(subdir, stored_configs, mgr):
```

Comment:
``mgr`` only used to log a message.  this method is in the ambari class... it already knows it's ambari